java/ql/src/semmle/code/java/NumberFormatException.qll

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
This is not needed as this is not a query but a library.

Reply:
Thanks @intrigus-lgtm for reviewing this PR. I've made the change.